src/screens/send/components/gas-fee-advanced/styles.ts

```diff
@@ -15,7 +15,7 @@ export default (theme: ITheme) =>
             borderRadius: BORDER_RADIUS,
             borderColor: 'gray',
             alignSelf: 'stretch',
-            backgroundColor: theme.colors.inputBackground,
```

Comment:
i think it's better to have different vars for input bg color and text colors. For dark theme  maybe they have the same value but on other themes it's not the case.